x/evm/types/interfaces.go

```diff
@@ -55,3 +57,12 @@ type EvmHooks interface {
 	// Must be called after tx is processed successfully, if return an error, the whole transaction is reverted.
 	PostTxProcessing(ctx sdk.Context, msg core.Message, receipt *ethtypes.Receipt) error
 }
+
+type (
+	LegacyParams = paramtypes.ParamSet
```

Comment:
This syntax seems to be new in go [1.19](https://go101.org/article/type-system-overview.html), called `Type Alias Declarations`.

Hadn't seen this before, nice! However, we're not using it on line 66. This could be 

```
		GetParamSetIfExists(ctx sdk.Context, ps LegacyParams)
```